### PR TITLE
 Introduce SystemEnvironmentUtils

### PR DESCRIPTION
**Pull request by krvikash**
## Release notes

(X) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by ebyhr — COMMENTED** [review]
Could you find other usages with `System.getenv`?

**Comment by krvikash** [comment]
Thanks, @ebyhr for the review. AC.

**Review by Praveen2112 — APPROVED** [review]

**Review by marcinsbd — APPROVED** [review]

**Review by ssheikin — APPROVED** [review]

**Comment by krvikash** [comment]
(rebased with master)

File: testing/trino-testing-services/src/main/java/io/trino/testing/SystemEnvUtils.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing;
+
+import static java.util.Objects.requireNonNull;
+
+public final class SystemEnvUtils
```
Comment (Praveen2112) [inline testing/trino-testing-services/src/main/java/io/trino/testing/SystemEnvUtils.java:18]: nit: Can we avoid abbreviations

**Comment by krvikash** [comment]
@Praveen2112 AC

**Comment by krvikash** [comment]
(rebased with master)

**Comment by krvikash** [comment]
(rebased with master)

**Pull request merged**